wp-content/themes/reconasia/assets/_scss/abstracts/_variables.scss

```diff
@@ -4,70 +4,61 @@
 
 /*----------  Colors  ----------*/
 $color-white-100: rgba(255, 255, 255, 1);
-$color-white-130: rgba(255, 255, 255, 0.3);
-$color-white-150: rgba(255, 255, 255, 0.5);
-
 $color-black-100: rgba(0, 0, 0, 1);
-$color-black-125: rgba(0, 0, 0, 0.25);
-$color-black-130: rgba(0, 0, 0, 0.3);
 
-$color-gray-1: rgba(51, 51, 51, 1);
+$color-dark-pattern-bg: $color-black-100;
+$color-text-link-primary-200: rgba(22, 126, 130, 1);
 
-$color-text-gray-05: rgba(240, 240, 240, 1);
-$color-text-gray-10: rgba(229, 229, 229, 1);
-$color-text-gray-30: rgba(173, 173, 173, 1);
-$color-text-gray-50: rgba(117, 117, 117, 1);
-$color-text-gray-70: rgba(69, 69, 69, 1);
-$color-text-gray-90: rgba(23, 23, 23, 1);
+$color-bg-light-100: $color-white-100;
+$color-bg-light-200: rgba(241, 245, 244, 1);
+$color-bg-light-300: rgba(230, 236, 235, 1);
+$color-bg-light-400: rgba(214, 224, 223, 1);
 
-$color-primary-base: rgba(0, 101, 166, 1);
-$color-primary-base-130: rgba(0, 101, 166, 0.3);
-$color-primary-base-180: rgba(25, 95, 143, 0.8);
-$color-primary-light: rgba(82, 146, 187, 1);
-$color-primary-dark: rgba(0, 72, 122, 1);
-$color-primary-dark-200: rgba(25, 95, 143, 1);
-$color-primary-darkest: rgba(36, 44, 48, 1);
+$color-bg-dark-100: rgba(13, 15, 18, 1);
+$color-bg-dark-120: rgba(0, 0, 0, 0.2);
+$color-bg-dark-200: rgba(30, 36, 41, 1);
+$color-bg-dark-290: rgba(30, 36, 41, 0.9);
+$color-bg-dark-300: rgba(39, 46, 51, 1);
+$color-bg-dark-400: rgba(48, 55, 61, 1);
 
-$color-gradient-light-end: rgba(245, 246, 247, 0.88);
-$color-gradient-dark-end: rgba(60, 72, 77, 1);
+$color-accent-light-primary-100: rgba(44, 101, 101, 1);
+$color-accent-light-primary-200: rgba(61, 130, 131, 1);
+$color-accent-light-primary-300: rgba(180, 208, 208, 1);
 
-$color-text-ltblue-80: rgba(165, 175, 181, 1);
-$color-text-ltblue-100: rgba(201, 207, 212, 1);
+$color-accent-dark-primary-400: rgba(106, 190, 176, 1);
 
-$color-bg-lt-blue: rgba(241, 244, 244, 1);
+$color-text-white-100: $color-white-100;
```

Comment:
Instead of using `text` in the variable name here, I would rename these to be `$color-white-100', `$color-black-140`, etc. They're more reusable that way.